benchmarks/generated_benchmark_suites.cmake

```diff
@@ -0,0 +1,59 @@
+################################################################################
+# Autogenerated by build_tools/benchmarks/generate_cmake_benchmark_suites.py   #
+# To update the benchmarks, modify the files in build_tools/benchmarks/suites/ #
+# and regenerate this file.                                                    #
+################################################################################
+
+################################################################################
+# Defines the required variables                                               #
+################################################################################
+iree_package_name(_PACKAGE_NAME)
+set(_ROOT_ARTIFACTS_DIR "${IREE_BINARY_DIR}/benchmark_suites")
+set(_MODEL_ARTIFACTS_DIR "${_ROOT_ARTIFACTS_DIR}/models")
+set(_IREE_ARTIFACTS_DIR "${_ROOT_ARTIFACTS_DIR}/iree")
+
+################################################################################
+# Below is generated by build_tools/benchmarks/suites/cmake_rule_generator.py  #
+################################################################################
+# Fetch the model from "https://storage.googleapis.com/iree-model-artifacts/mobilenet_v2_1.0_224.tflite"
+add_custom_command(
```

Comment:
Even though we're generating it, it might still be nice to have a function for this custom_command + custom_target

Reply:
Will do later.